include/native/parallel_execution_native.h

```diff
@@ -757,7 +756,7 @@ template <typename ... InputIterators, typename Identity,
 auto parallel_execution_native::map_reduce(
     std::tuple<InputIterators...> firsts, 
     std::size_t sequence_size,
-    Identity && identity,
+    Identity && identity_,
```

Comment:
trailing underscores in identifiers is reserved in GrPPI for class data members.